src/components/Form/FormSelect.react.js

```diff
@@ -3,14 +3,22 @@
 import * as React from "react";
 import cn from "classnames";
 
-type Props = {| +children?: React.Node, +className?: string |};
+type Props = {|
+  +children?: React.Node,
+  +className?: string,
+  +onChange?: Function,
```

Comment:
Let's make this type a little more specific i.e. something similar to `(SyntheticInputEvent<some specific type>) => bool or void` 

Obviously that's not quite 100% right but that's what I can recall off the top of my head - [this link](https://flow.org/en/docs/types/functions/#toc-function-types) should help